std/codec.proto

```diff
@@ -134,6 +138,19 @@ message Message {
     cosmos_sdk.x.staking.v1.MsgDelegate                         msg_delegate                      = 15;
     cosmos_sdk.x.staking.v1.MsgBeginRedelegate                  msg_begin_redelegate              = 16;
     cosmos_sdk.x.staking.v1.MsgUndelegate                       msg_undelegate                    = 17;
+    cosmos_sdk.x.ibc.connection.v1.MsgConnectionOpenInit        msg_connection_open_init          = 18;
+    cosmos_sdk.x.ibc.connection.v1.MsgConnectionOpenTry         msg_connection_open_try           = 19;
+    cosmos_sdk.x.ibc.connection.v1.MsgConnectionOpenAck         msg_connection_open_ack           = 20;
+    cosmos_sdk.x.ibc.connection.v1.MsgConnectionOpenConfirm     msg_connection_open_confirm       = 21;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelOpenInit              msg_channel_open_init             = 22;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelOpenTry               msg_channel_open_try              = 23;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelOpenAck               msg_channel_open_ack              = 24;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelOpenConfirm           msg_channel_open_confirm          = 25;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelCloseInit             msg_channel_close_init            = 26;
+    cosmos_sdk.x.ibc.channel.v1.MsgChannelCloseConfirm          msg_channel_close_confirm         = 27;
+    cosmos_sdk.x.ibc.channel.v1.MsgPacket                       msg_packet                        = 28;
+    cosmos_sdk.x.ibc.channel.v1.MsgAcknowledgement              msg_acknowledgement               = 29;
+    cosmos_sdk.x.ibc.transfer.v1.MsgTransfer                    msg_transfer                      = 30;
```

Comment:
This whole file is going away and we're not using `oneof`s so this can be deleted. See #6081 
```suggestion
```

Reply:
I'd prefer to leave it as is and remove them completely when the migration is done. If that is ok with you